src/app/AppLayout/AppLayout.tsx

```diff
@@ -242,7 +229,7 @@ const AppLayout: React.FunctionComponent<AppLayoutProps> = ({ children }) => {
   );
 
   const mobileOnSelect = React.useCallback(
-    (selected) => {
+    (_selected) => {
```

Comment:
We can use `_` here?

Reply:
Actually, any guides between using `_` and `_some_var`?

Reply:
Well, just there is a eslint rule unused-imports/no-unused-vars which warns if there are any unused vars, but these can be bypassed if the variable matches the regex in the .eslintrc file. Here is the snippet.

```
		"unused-imports/no-unused-vars": [
			"warn",
			{ "vars": "all", "varsIgnorePattern": "^_", "args": "after-used", "argsIgnorePattern": "^_" }
    ]
```

Reply:
I don't think I correctly chose the way I wanted to use `_selected` here instead of `_`. I think in the future `_somevar` should be used if the variable is a an argument of a function that comes from a function that we created in the cryostat-web, but if it comes as a prop (e.g. here from `mobileOnSelect` which is a function handler used as a callback for Patternfly's `Nav` component (**which we didn't create**)), then we should use `_`. I will change that here.